<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Sys_Database_Dia</class>
 <widget class="QDialog" name="Sys_Database_Dia">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>350</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>420</width>
    <height>350</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>430</width>
    <height>370</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Enter Connection Parameters</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources/system_sys_resource.qrc">
    <normaloff>:/prom_icon</normaloff>:/prom_icon</iconset>
  </property>
  <widget class="QGroupBox" name="ConGroupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>407</width>
     <height>289</height>
    </rect>
   </property>
   <property name="title">
    <string>Connection Parameters</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>41</y>
      <width>377</width>
      <height>26</height>
     </rect>
    </property>
    <layout class="QHBoxLayout">
     <item>
      <widget class="QLabel" name="hostnameLabel">
       <property name="text">
        <string>Hostname</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="editHostname"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>170</y>
      <width>377</width>
      <height>26</height>
     </rect>
    </property>
    <layout class="QHBoxLayout">
     <item>
      <widget class="QLabel" name="passwordLabel">
       <property name="text">
        <string>Password </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="editPassword">
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>127</y>
      <width>377</width>
      <height>26</height>
     </rect>
    </property>
    <layout class="QHBoxLayout">
     <item>
      <widget class="QLabel" name="usernameLabel">
       <property name="text">
        <string>Username</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="editUsername"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>256</y>
      <width>377</width>
      <height>30</height>
     </rect>
    </property>
    <layout class="QHBoxLayout">
     <item>
      <widget class="QLabel" name="TableDirLabel">
       <property name="text">
        <string>Tables Directory</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="editTableDir"/>
     </item>
     <item>
      <widget class="QPushButton" name="browseTableDirButton">
       <property name="text">
        <string>Browse</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>84</y>
      <width>377</width>
      <height>26</height>
     </rect>
    </property>
    <layout class="QHBoxLayout">
     <item>
      <widget class="QLabel" name="databaseLabel">
       <property name="text">
        <string>Database </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="editDatabase"/>
     </item>
    </layout>
   </widget>
   <widget class="QCheckBox" name="checkBox_save_pass">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>200</y>
      <width>91</width>
      <height>19</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>For security reasons the saving of the password is not recommended!</string>
    </property>
    <property name="text">
     <string>Save password</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>381</width>
      <height>24</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="driverlabel">
       <property name="text">
        <string>Driver</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QComboBox" name="drivercombo">
       <item>
        <property name="text">
         <string>QPSQL</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>320</y>
     <width>407</width>
     <height>30</height>
    </rect>
   </property>
   <layout class="QHBoxLayout">
    <item>
     <spacer>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>191</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="okButton">
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="cancelButton">
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="resources/system_sys_resource.qrc"/>
 </resources>
 <connections/>
</ui>
